kops/kops.go

```diff
@@ -156,11 +145,8 @@ func (k *KopsCmd) UpdateCluster(cluster clusteroperatorv1alpha1.KopsConfig) erro
 		return nil
 	}
 
-	kopsCmd := "/usr/local/bin/" +
-		"docker run" +
-		utils.GetDockerEnvFlags(k.envs) +
-		" soheileizadi/kops:v1.0" +
-		" update cluster " +
+	kopsCmd := ".bin/" +
+		"kops update cluster " +
```

Comment:
make the cmd a configurable path. The path won't have to be updated in all the places when we change where the binary is pulled from ie. /usr/local/bin/kops, .bin/kops, /usr/local/bin/kops-vX.X.X

Reply:
latest commit addresses this